tetrateci/create_istio_release.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+cd ..
+git clone https://github.com/istio/release-builder --depth=1
+envsubst < ./istio/tetrateci/manifest.yaml.in > ./release-builder/manifest.yaml
+cd release-builder
+cp -r ../istio .
```

Comment:
Huh..is this for volume mounting into the container?

Reply:
Yes